README.md

```diff
@@ -28,6 +28,9 @@ More complete install documentation is available on the PostgreSQL [wiki](https:
 
 **Note:** If you're setting up the project on Mac OSX, we recommend using [homebrew](http://brew.sh) to install PostgreSQL.
 
+### Install Memcached
```

Comment:
You'll need memcached installed to `pip install -r requirements.txt` (pylibmc needs it)